python.d/README.md

```diff
@@ -2007,6 +2007,61 @@ local:
 
 ---
 
+# powerdns_recursor
+
+Module monitor powerdns recursor performance and health metrics.
+
+Following charts are drawn:
+
+1. **Questions In**
```

Comment:
Is this whole 'Questions'/'Answers' thing the actual terms used by PowerDNS?  If so, it's fine to keep them, but if not, please either:

1. Update this to match up with the PowerDNS terminology.

or:

2. Use the actual DNS terminology ('Queries'/'Responses').

Reply:
https://doc.powerdns.com/recursor/metrics.html#gathered-information

Yep questions and answers are the terms used by the pdns-recursor API

Reply:
Huh, that's a really odd choice of names.  Technically, it is semantically correct, it's just inconsistent with pretty much every other piece of DNS software in existence.